playground/package.json

```
@@ -1,5 +1,6 @@
 {
   "name": "playground",
+  "private": true,
```

ESLint

[error] Expected object keys to be in specified order. 'private' should be after 'version'.

(jsonc/sort-keys)